src/virtual-wan/azext_vwan/_params.py

```diff
@@ -61,6 +61,9 @@ def load_arguments(self, _):
         c.argument('p2s_vpn_gateway', help='Name or ID of a P2S VPN gateway.', validator=get_network_resource_name_or_id('p2s_vpn_gateway', 'P2sVpnGateways'))
         c.argument('vpn_gateway', help='Name or ID of a VPN gateway.', validator=get_network_resource_name_or_id('vpn_gateway', 'vpnGateways'))
 
+    with self.argument_context('network vhub get-effective-routes') as c:
+        c.argument('virtual_wan_resource_type', options_list='--resource-type')
```

Comment:
Is there an enum for this argument?

Reply:
It's string and help tells users which values they can use.